payment-request/payment-request-canmakepayment-method.https.html

```diff
@@ -88,15 +88,29 @@
   const unsupportedMethods = [
     "this-is-not-supported",
     "https://not.supported",
-    "basic-card?not-really",
-    "basic-card://not-ok",
-    "basic card",
-    "/basic card/",
-    "BaSicCarD",
-    "BASIC-CARD",
-    " basic-card ",
-    "this is not supported",
-    " ",
+    "e",
```

Comment:
Maybe we should change: 
s/this-is-not-supported/this-is-not-supported-but-valid
https://not.supported/https://not.supported.but.valid

Reply:
fixed comment above... people might get confused with "supported" vs "invalid"